.github/workflows/checks.yml

```diff
@@ -131,7 +131,7 @@ jobs:
         run: npm ci
       - name: Run Playwright tests
         run: npm run e2e:ci
-      - uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@v4
```

Comment:
is this change related?

Reply:
No, it was used to test this fix https://github.com/dfinity/gix-components/pull/559. And will be reset after the fix is merged.